tails_files/install.sh

```diff
@@ -162,13 +162,20 @@ if [ -f $XSESSION_RC ]; then
     fi
 fi
 
+# remove previous NetworkManager hook if present. it runs later than we'd like
+OLD_NM_HOOK=$TAILSCFG/custom-nm-hooks/99-tor-reload.sh
+if [ -f $OLD_NM_HOOK ]; then
+    rm -f $OLD_NM_HOOK > /dev/null 2>&1
+    rm -f $INSTALL_DIR/99-tor-reload.sh > /dev/null 2>&1
+fi
```

Comment:
Also remove old script from `/etc/NetworkManager/dispatcher.d/`, since it may be there already as configured by a prior version of the init script.

Might be wise to drop the conditional block altogether: we want to ensure removal of the old script no matter what.